monitoring/grafana/dashboards/cephfs-overview.json

```diff
@@ -77,7 +78,12 @@
       "pointradius": 5,
       "points": false,
       "renderer": "flot",
-      "seriesOverrides": [],
+      "seriesOverrides": [
+        {
+          "alias": "/.*Reads/",
+          "transform": "negative-Y"
```

Comment:
Interesting approach. I tried this once and had the impression that I was losing detail due to the increase in the vertical span of the graph. I haven't tried with recent Grafana releases: is the horizontal axis kept centered or it scales from bottom to top?

Reply:
One certainly looses granularity but on the other hand gains more clarity which direction data flows. I'd argue that these panels are rarely used to get accurate measurements of say an ingress rate (though that is certainly still an option) but rather to see if there's a heavy read or write workload currently stressing the cluster.

The y-axis still scales depending on the data, i.e. with no Reads in the above panel the 0 point should be at the bottom of the panel.